composer.json

```diff
@@ -30,7 +32,8 @@
     },
     "suggest": {
         "netgen/ibexa-site-api": "Boost your site-building productivity with Ibexa CMS",
-        "ibexa/solr": "Supports advanced capabilities with Ibexa search API"
+        "ibexa/solr": "Supports advanced capabilities with Ibexa search API",
+        "ibexa/elasticsearch": "Needed for layouts indexer"
```

Comment:
"Supports advanced capabilities with Ibexa search API", samse as for Solr engine.